cmd/thanos/sidecar.go

```diff
@@ -247,12 +250,46 @@ func runSidecar(
 			return errors.Wrap(err, "setup gRPC server")
 		}
 
+		examplarSrv := exemplars.NewPrometheus(conf.prometheus.url, c, m.Labels)
+
+		infoSrv := info.NewInfoServer(
+			component.Sidecar.String(),
+			func() []labelpb.ZLabelSet {
+				return promStore.LabelSet()
+			},
+			func() *infopb.StoreInfo {
+				mint, maxt := promStore.Timestamps()
+				return &infopb.StoreInfo{
+					MinTime: mint,
+					MaxTime: maxt,
+				}
+			},
+			func() *infopb.ExemplarsInfo {
+				// Currently Exemplars API does not expose metadata such as min/max time,
+				// so we are using default minimum and maximum possible values as min/max time.
+				return &infopb.ExemplarsInfo{
+					MinTime: time.Unix(math.MinInt64/1000+62135596801, 0).UTC().Unix(),
+					MaxTime: time.Unix(math.MaxInt64/1000-62135596801, 999999999).UTC().Unix(),
+				}
```

Comment:
We can use global variable for the two values to avoid calculating them each time.

Reply:
@hitanshu-mehta We are still calling `.Unix()` every time. Can't we just initialize `v1.MinTime` and `v1.MaxTime` as int64? Are they used anywhere else?

Reply:
Currently, they are not being used anywhere else. Agree, I should've used int64. I'll correct it. Thank you!

Reply:
The thing is we can have kind of constants somewhere in global level. Can we do that? It's still to be imp[roved?